src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -250,6 +256,76 @@ public string GetLanguage()
             return node == null ? null : node.Value;
         }
 
+        /// <summary>
+        /// Build action groups
+        /// </summary>
+        public IEnumerable<ContentFilesEntry> GetContentFiles()
+        {
+            var ns = MetadataNode.GetDefaultNamespace().NamespaceName;
+
+            foreach (var filesNode in MetadataNode
+                .Elements(XName.Get(ContentFiles, ns))
+                .Elements(XName.Get(Files, ns)))
+            {
+                var include = GetAttributeValue(filesNode, "include");
+
+                if (include == null)
+                {
+                    // Invalid include
+                    var message = Strings.FormatInvalidNuspecEntry(
```

Comment:
I might be missing something. Does 'Strings.FormatInvalidNuspecEntry' handle localization?